vault/token_store.go

```diff
@@ -1500,7 +1592,7 @@ func (ts *TokenStore) handleTidy(ctx context.Context, req *logical.Request, data
 		// more and conclude that accessor, leases, and secondary index entries
 		// for this token should not exist as well.
 		if te == nil {
-			ts.logger.Info("deleting token with nil entry", "salted_token", saltedID)
+			ts.logger.Info("deleting token with nil entry")
```

Comment:
Any reason you took out the extra info?

Reply:
I had removed it since I did not have direct access to the obfuscatedID in the function anymore as the logic was abstracted into `lookupTokenNonLocked`. But, I guess there is no reason to remove a functionality just for making code look a little neat. Added it back again 👍.